### DISCUSSION TITLE
if you are in search of star 👇

### DISCUSSION THREAD

**Discussion by csthenry** [community/community#27244] [Other Feature Feedback, Questions, & Ideas]
please leave a star here 🙏🙏 : **(removed by moderator)**
i will help others in search of star,
btw thanks in advance

~ add a star and add leave your repo link here.

**Comment by ghost** [comment]
make Readme file in English

**Reply by csthenry** [reply]
i can not think what you say...

**Reply by csthenry** [reply]
my repo readme?

**Reply by ghost** [reply]
yes

**Reply by csthenry** [reply]
I may make readme file in English. thank u.

**Reply by ghost** [reply]
you are welcome

**Comment by csthenry** [comment]
**(removed by moderator)**

**Reply by chienhsiang-hung** [reply]
I like **(removed by moderator)** this one! great job!
here is mine
**(removed by moderator)**
and thank you in advance @csthenry :)

**Reply by csthenry** [reply]
thank u, If you can, please leave a star for my **(removed by moderator)**, i want an achievement,  thank you

**Reply by Karkatov** [reply]
Thank u:)

**Comment by Karkatov** [comment]
I will mutually add your repository to favorites, here is mine: **(removed by moderator)**

**Reply by csthenry** [reply]
add my repository to favorites,before.

**Reply by Karkatov** [reply]
👍

**Reply by csthenry** [reply]
@Karkatov done

**Comment by tuves** [comment]
Hi @csthenry, @Karkatov, @chienhsiang-hung, @GAbdulWahab,

We're glad you're enjoying our star system. However, we want to ensure the community is a space for users to ask questions without overwhelming them with other conversations. Please refrain from self-promotion conversations such as these. For additional guidance, please review our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/). Thank you for helping us maintain a productive and tidy community for all our members.

**Reply by alinakhay** [reply]
Hi! Lets exchange the stars and follow each other https://github.com/alinakhay/Text_Classification_with_NLP_Tf-Idf_vs_Word2Vec_vs_BERT

**Comment by Maltered** [comment]
please, stars this one, I'm doing a competition with my friend and I need to win 🙏, i exchange  https://github.com/Maltered/freemind-Project

**Reply by ganochenkodg** [reply]
Done, waiting for a star from you https://github.com/ganochenkodg/qdrant-operator :)

**Comment by queenofcorgis** [comment]
We have closed and locked this post due to excessive Code of Conduct violations.
